onnxruntime/python/tools/transformers/fusion_bart_attention.py

```diff
@@ -301,8 +517,10 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
         # 4) Decoder cross attention with two_root_inputs=True and qk_nodes=qk_nodes_1
         # 5) Decoder cross attention with past with three_root_inputs=True and qk_nodes=qk_nodes_1
         encoder_attention = one_root_input and qk_nodes == qk_nodes_1
-        decoder_attention = one_root_input and qk_nodes == qk_nodes_2
+        decoder_attention = one_root_input and qk_nodes in (qk_nodes_2, qk_nodes_2_openai)
         decoder_attention_with_past = encoder_attention and past_k and past_v
```

Comment:
Can this be simplified as follows?

```suggestion
        decoder_attention_with_past = (encoder_attention if not model_impl_openai else decoder_attention) and past_k and past_v
```

Reply:
This is what I had previously, the RUFF checks in the lintrunner seems to complain about this.

Reply:
For the error code that is returned, you can put a comment after the line so that the linter ignores this.

Example:
```
decoder_attention_with_past = (encoder_attention if not model_impl_openai else decoder_attention) and past_k and past_v  # noqa: <error code here>
```